src/animation.js

```diff
@@ -76,10 +82,21 @@ exports.requestAnimationFrame = function(callback, node) {
 /**
 * Cancels a requested animation callback with the specified id.
 *
+* @param {Number} id - The identifier of an animation request we wish to cancel.
+* @deprecated since 2.7.0
+* @public
+*/
+exports.cancelRequestAnimationFrame = function(id) {
```

Comment:
Actually this won't work if we're using the native rAF and cAF methods as the scope will be incorrect. Vote to leave it as-is.